mmv1/products/iamworkforcepool/WorkforcePoolProvider.yaml

```diff
@@ -244,4 +244,24 @@ properties:
         description: The client ID. Must match the audience claim of the JWT issued
           by the identity provider.
         required: true
-
+      - !ruby/object:Api::Type::NestedObject
+        name: 'webSsoConfig'
+        description: |
```

Comment:
Will this field be populated for existing configs? It may need the `default_from_api: true` so Terraform will default to the API's value if not in config. @zli82016

Reply:
This is a good point. @misterpoe , I noticed you already added `default_from_api: true`. I wonder if it is needed or not. If the `webSsoConfig` is not provided, will there be default value from API?

Reply:
Yes, there will be default value from API. Thanks for bringing this up, made the change.

Reply:
@misterpoe , can you please do not add `default_from_api: true` to the new field and then add another test without the new field `webSsoConfig`? If the test passes, then `default_from_api: true` is not needed. I tried to run the test locally, but I do not have the required premission.